Update benchmarking numbers

kimishpatel opened the pull request:
Summary:
ATT

Created from CodeHub with https://fburl.com/edit-in-codehub

Reviewed By: lucylq

Differential Revision: D55817614

mergennachin reviewed (approved):
In the instructions below, can you add an instruction on how to reproduce with different group sizes?

kimishpatel committed:
Update benchmarking numbers (#2881)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/2881

ATT

Created from CodeHub with https://fburl.com/edit-in-codehub

Reviewed By: mergennachin, lucylq

Differential Revision: D55817614
